svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -464,7 +410,7 @@ public function addSlickPageHeader()
 
         $ldJsonPlugin = (object) [
             '@type' => 'Plugin',
-            'version' => '1.2.6',
+            'version' => '1.2.7',
```

Comment:
Let's bump this to `1.3.0` since this is a more substantial change than just a 'patch' release.